plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -149,7 +149,7 @@
          on to the appropriate Host (virtual host).
          Documentation at /docs/config/engine.html -->
 
-    <Engine name="Catalina" defaultHost="localhost">
+    <Engine name="Catalina" defaultHost="localhost" jvmRoute="${exo.cluster.node.name}">
```

Comment:
What is the interest to put this variable here while the jvmRoute can be specified as a standard system property ? (read https://tomcat.apache.org/tomcat-7.0-doc/config/engine.html)
How does the Tomcat Engine behave when the ${exo.cluster.node.name} var is empty in a non cluster scenario ?